### ISSUE TITLE
Consider a Patrons Scheme

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#1942]
For consideration - Pat and Rich to write paper for Trustees as initial action

Idea: in return for an annual fee, researchers get a number of services, some on Github as Registered Researcher services, plus others: i.e.
* advert-free

E4 U3 S2 C5: P14

Update: 31/1/2020: scheme to not be associated with "Registered Researchers" but be a "Patron's scheme" designed to raise funds through regular giving.

**Comment by PatReynolds** [comment]
Draft discussion document for Trustees: https://docs.google.com/document/d/1QPxMw3fdjdG4ox8s8AuiTCRnTGkpGBsEcaRSoaIQdFY/edit?usp=sharing  
Have I missed anything?

**Comment by edickens** [comment]
Planned features:-  Researcher can add a searchable correction to, but not change, an entry.  (Record should be Entry.)
:- Researcher can view (and download?) the image.  This needs thought because of the terms and conditions under which we got the images, but if they have paid, then I think that they need to have this option.
For the charge I would go for £10 by credit card for one month, or by direct debit £20 a year/£2 a month.  Avoid the 99p.  Would this be for all FreeUKGenealogy?  Viewing FreeBMD and FreeCEN images could be chargeable.
The big advantage of registered researchers is that we have name and email address.  So any breaches of terms and conditions are traceable.  All images that have been scanned for FreeREG have a marker in the top left corner.  When images are shown to a researcher they could have a watermark or something to identify them.  Showing one image at a time does not help anyone who wants to "steal" our images, but is very useful to check a transcription or save in their family tree.

**Comment by PatReynolds** [comment]
Would there be a single log in

**Comment by PatReynolds** [comment]
After discussion with Trustees and advisory board 28/9: 
inclusion of a paid-for "no adve rt service would need to introduce carefully so that not confused with our ongoing commitment to free access.  Would need to decide if this was a separate category to RR.  Could be called "Patrons".  Other additional benefit could be being asked to comment on "what next" features, and get to beta test them.

Single login not as complex as for volunteers as all one set of permissions, and no relationships.  Could help to encourge use of our other websites.

**Comment by edickens** [comment]
I would suggest that we do not want yet another category.  RR (Registered Researcher) should do.  But it does need to be a new list of "UserIDs" so that it covers all the projects.  And Researcher needs to be removed from the list of Roles in the FreeREG UserID.
I am not sure that people want to pay just to remove the Advertisements.  Ours are not very intrusive.  Does anyone have evidence?  Nobody has mentioned this to me.
Our log in is not very complex.  But we may perhaps ask fo less information than we do for transcribers.  We must verify the email address as we need to be sure that the person is genuine and understands why they have registered and abides by our T&C.

**Comment by PatReynolds** [comment]
Agreed, two categories needed, Registered Researchers (free, this story) and "new term" (paid, new story needed).

**Comment by PatReynolds** [comment]
Revised document for review: https://docs.google.com/document/d/1JnKsYWzf39KOY5bqbmNfz8ANskLA2w_RMe9NVfxT_KE/edit?usp=sharing

**Comment by edickens** [comment]
I must go with what the Trustees have agreed.  But I really don't think that provided the present search facilities are free, that we will be seen as no longer free if we ask for a contribution for extra facilities, especially if we have to maintain a database of registered users in order to control access to images and such.
Also, our advertising is not intrusive.  I don't think that you will get many people paying to have them removed, and yet it will involve accounting to provide the facility.
Is some market research needed?

**Comment by richpomfret** [comment]
@richpomfret to review.

**Comment by PatReynolds** [comment]
New version of discussion document made (v.3) to reflect change of name

**Comment by richpomfret** [comment]
@PatReynolds I think we can now close this task and create the 'Create Patron' Epic. I've made a small start on a list of tasks/schedule for getting the simplistic version of this up and running.

**Comment by PatReynolds** [comment]
Agreed.

**Comment by PatReynolds** [comment]
Note for when this is re-activated: will need a cookie on the user's website, and way of re-implementing after user has cleared all cookies for some reason.

Cookie could be used to customise experience (e.g. external sites always open in a new tab)
